app/utils.py

```diff
@@ -10,14 +10,20 @@
 from sentry_sdk.integrations.logging import LoggingIntegration
 
 from app.config import settings
-from app.schemas import LocationFull, ProductFull
+from app.models import Location, Product
 
 logger = get_logger(__name__)
 
 
+def get_user_agent() -> str:
```

Comment:
this doesn't seem linked to typing

Reply:
It's necessary in order to use new `openfoodfacts` python package which improves typing.